### DISCUSSION TITLE
安装小雅emby报错 [ERROR] config.mp4 不存在，请下载此文件并解压

### DISCUSSION THREAD

**Discussion by eihana** [monlor/docker-xiaoya#275] [Q&A]
第一次安装 当中不小心把终端关了.
之后再安装时候 清楚了config 重新定义了路径
然后安装emby全家桶时候就报错了 我看了说清理阿里网盘 但是我网盘是空的
换了token还是一样错

[INFO] 小雅配置目录通过小雅容器获取
[INFO] 已读取小雅Alist配置文件路径：etc/xiaoya (默认不更改回车继续，如果需要更改请输入新路径)
CONFIG_DIR:
[INFO] 读取配置目录中...
[INFO] 媒体库目录通过 emby_config.txt 获取
[INFO] 已读取媒体库目录：/mnt/disk/xiaoya (默认不更改回车继续，如果需要更改请输入新路径)
MEDIA_DIR:
[INFO] docker0 的 IP 地址是：172.17.0.1
[INFO] 测试xiaoya的联通性...
Location: https://cn-beijing-data.aliyundrive.net/DZnxNDmz%2F189426850%2F648ca9cef1ba5a23502344f8951e7e20b6237710%2F64970917db671c513fa943b1ba9b5784f2add83d?callback=eyJjYWxsYmFja1VybCI6Imh0dHA6Ly9iajI5LmFwaS1ocC5hbGl5dW5wZHMuY29tL3YyL2ZpbGUvZG93bmxvYWRfY2FsbGJhY2siLCJjYWxsYmFja0JvZHkiOiJodHRwSGVhZGVyLnJhbmdlPSR7aHR0cEhlYWRlci5yYW5nZX1cdTAwMjZidWNrZXQ9JHtidWNrZXR9XHUwMDI2b2JqZWN0PSR7b2JqZWN0fVx1MDAyNmRvbWFpbl9pZD0ke3g6ZG9tYWluX2lkfVx1MDAyNnVzZXJfaWQ9JHt4OnVzZXJfaWR9XHUwMDI2ZHJpdmVfaWQ9JHt4OmRyaXZlX2lkfVx1MDAyNmZpbGVfaWQ9JHt4OmZpbGVfaWR9XHUwMDI2cGRzX3BhcmFtcz0ke3g6cGRzX3BhcmFtc31cdTAwMjZ2ZXJzaW9uPSR7eDp2ZXJzaW9ufSIsImNhbGxiYWNrQm9keVR5cGUiOiJhcHBsaWNhdGlvbi94LXd3dy1mb3JtLXVybGVuY29kZWQiLCJjYWxsYmFja1N0YWdlIjoiYmVmb3JlLWV4ZWN1dGUiLCJjYWxsYmFja0ZhaWx1cmVBY3Rpb24iOiJpZ25vcmUifQ%3D%3D&callback-var=eyJ4OmRvbWFpbl9pZCI6ImJqMjkiLCJ4OnVzZXJfaWQiOiJiNTRiMzFhN2JkY2U0YjFhOGI0YzQ0MGYxYTdmZThjYyIsIng6ZHJpdmVfaWQiOiI4ODMwOTc5NzMiLCJ4OmZpbGVfaWQiOiI2NzJkMTA0YzhiYzk2NjE0Zjk5NDQ1ODM5OWMwNGExMmUzZDViYTNmIiwieDpwZHNfcGFyYW1zIjoie1wiYXBcIjpcIjc2OTE3Y2NjY2Q0NDQxYzM5NDU3YTA0ZjYwODRmYjJmXCJ9IiwieDp2ZXJzaW9uIjoidjMifQ%3D%3D&di=bj29&dr=883097973&f=672d104c8bc96614f994458399c04a12e3d5ba3f&pds-params=%7B%22ap%22%3A%2276917ccccd4441c39457a04f6084fb2f%22%7D&response-content-disposition=attachment%3B%20filename%2A%3DUTF-8%27%27README.md&security-token=CAISvgJ1q6Ft5B2yfSjIr5flf9funopO5q%2BJY3bJnG4UWfZ7gbya0zz2IHhMf3NpBOkZvvQ1lGlU6%2Fcalq5rR4QAXlDfNSS2FAGPq1HPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLs0ARrv4r1fMUqW2X543AUgFLhy2KKMPY99xpFgh9a7j0iCbSGyUu%2FhcRm5sw9%2Byfo34lVYneU9MCShfOi4IClLcc%2BmqdsRIvJzWstJ7Gf9LWqChvSgk4TxhhcNFKSTQrInFCB0%2BcRObJl16iZh5w2fXtuMkagAGwoDziB2fuNfrLws3VFNTPwgeJYmCkXBHXZyRW%2BTuhzS8NGnToKuAr5ZvYsZAYZ3lPqCcw7vbLbqGvN%2BuNjgwYOtTSmbCP4tuRB7iPpJfSYGnBlE3aSr3FcxrBEBGpoRSBAjNGI%2BKDjiWAuV5Mr%2FoY8ieiRgwMge7kfVfH%2B3YMySAA&u=b54b31a7bdce4b1a8b4c440f1a7fe8cc&x-oss-access-key-id=STS.NTP4mZsUoQmbaPxxnAUyWny11&x-oss-expires=1731020842&x-oss-signature=3SPAftGs7lP%2Fj9CWjTgbkdnDK9vhbhs2kbEZJGsEbno%3D&x-oss-signature-version=OSS2
x-oss-request-id: 672D104F7A795E36388A2A92
x-oss-object-type: Multipart
x-oss-hash-func: SHA-1
x-oss-hash-value: 780D633F85BFF28B4E47422B23CF98BB64D913FA
x-oss-hash-crc64ecma: 17125971843307411358
x-oss-storage-class: Standard
x-oss-server-time: 32
x-oss-qos-delay-time: 110
[INFO] 连接小雅地址为 http://127.0.0.1:5678
[INFO] 磁盘容量：230G
[INFO] 开始下载解压...
Untagged: xiaoyaliu/glue:latest
Deleted: sha256:e41875c77ae88720f93a201902c7ae3c5e94d81cc383cfd905c765847243913e
Deleted: sha256:ee00016dcf47a53534415db6dd0eaa8f5982c7952dec356ad69967d4d88ff9cd
Deleted: sha256:19e9d93201724a6667ea67fc5b3db0ceb0933752d4d7f0c3f289fa202be9efea
Deleted: sha256:6a727eef94a70316833f601f1493fcc30b262a91084bd8b2ff73d91bb311e444
Deleted: sha256:6a98c5a41b751908a39144484e7565984ba0f86fdc15613f4ca4a50a033fffbc
Deleted: sha256:5498e8c22f6996f25ef193ee58617d5b37e2a96decf22e72de13c3b34e147591
latest: Pulling from xiaoyaliu/glue
bccd10f490ab: Pull complete 
d10ba5459717: Pull complete 
547c4f31f115: Pull complete 
a36d51af957b: Pull complete 
e80762760778: Pull complete 
Digest: sha256:4d0523ff4efe06e594b4e00cf6ef9e92c70743c6192de0770b1b5f1f0a18c0ac
Status: Downloaded newer image for dockerhub.anzu.vip/xiaoyaliu/glue:latest
dockerhub.anzu.vip/xiaoyaliu/glue:latest
[INFO] xiaoyaliu/glue:latest 镜像拉取成功！
docker: Error response from daemon: create etc/xiaoya: "etc/xiaoya" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
See 'docker run --help'.
[INFO] docker0 的 IP 地址是：172.17.0.1
[INFO] 设置目录权限...
[INFO] 这可能需要一定时间，请耐心等待！
[INFO] 下载解压完成！
[INFO] docker0 的 IP 地址是：172.17.0.1
[ERROR] config.mp4 不存在，请下载此文件并解压！

**Comment by wwwcisco** [comment]
我和你的错误一样
